cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -110,6 +122,18 @@ void DrawableSubmap::Update(
     // for this submap.
     UpdateTransform();
   }
+  visibility_->setName(QStringLiteral("%1-%2.%3")
+                           .arg(trajectory_id_)
+                           .arg(submap_index_)
+                           .arg(metadata_version_));
+  visibility_->setDescription(
```

Comment:
Too bad you don't like it, I really do :-) Since all of the submaps overlap (because laser fans are always inserted in two most recent submaps), toggling one individual submap off at the time is not visually striking - it just makes one part of the map a little darker. By also toggling the neighbouring submaps at the same moment, we get the visual effect that we've really toggled a nice chunk of the map. However, if you want it removed, I can take it out.

Also, I have another idea - instead removing it altogether, this code construct (SubmapsDisplay receives a signal from one Submap and toggles others) can be modified - not to check if Ctrl was pressed and toggle the neighbours, which I admit is a bit hack-ish, but check instead if any other submaps were earlier shift-click selected in the submap list, and to also toggle them. Let me know what you think.